backend/route/app.js

```diff
@@ -0,0 +1,6 @@
+const sampleroute = require('./SampleRoute')
```

Comment:
name this file as index.js and make required changes and also document the whole flow as a Development guide in this folder Readme

Reply:
I am not familiar with the development guide part, could you link me to some examples that I could refer to?

Reply:
you can take help from here https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/tree/main/hotelontouch
it is a guide for React App please document your thing as per your file structure